test/es7/rlexe.xml

```diff
@@ -92,4 +92,12 @@
       <tags>exclude_xplat</tags>
     </default>
   </test>
+    <test>
+    <default>
+      <files>PromiseRejectionTracking.js</files>
+      <compile-flags>-TrackRejectedPromises -args summary -endargs -nodeferparse</compile-flags>
+      <baseline>PromiseRejectionTracking.baseline</baseline>
+      <tags>exclude_jshost</tags>
```

Comment:
>exclude_jshost [](start = 12, length = 14)

Great. Thanks for covering with detailed test cases. @taylor.woll@microsoft.com should we be implementing this in jshost just to get consistency.

Reply:
I was the one who added the tag.  I think this is something we should do in the near future, it's just a matter of scheduling.

/cc @boingoing